### PR TITLE
Update dashboard to not skip heading levels

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
The heading levels on the dashboard should progress without skipping levels, so this adjusts them in a few places to not do that and fix the styling when necessary.

## Testing done
Ran axe browser extension, verified that heading level error was limited to just the footer for several dashboard states

## Screenshots
No visual changes:
Screen Shot 2019-06-25 at 2 02 35 PM
Screen Shot 2019-06-25 at 2 02 30 PM

## Acceptance criteria
- [x] Heading levels aren't skipped

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -685,9 +685,9 @@
     to-fast-properties "^2.0.0"
 
 "@department-of-veterans-affairs/formation-react@^4.7.0":
-  version "4.7.0"
-  resolved "https://registry.yarnpkg.com/@department-of-veterans-affairs/formation-react/-/formation-react-4.7.0.tgz#669047353992e1803c096407e348940fe46d1ed8"
-  integrity sha512-ttCpkHaSjzshsfEBXiTMhWsVhpLnX6D6GB6x99owagAaJI2JIZy71QZbO4ZuEZvpAaVGmbqzSjIdM9HfGFBezg==
+  version "4.8.0"
```
Comment (jbalboni) [inline yarn.lock]: This is to pull in the change where I can set the `level` prop in the alert box and control which `h` element is used.

Comment (erikphansen) [reply yarn.lock]: I'm not seeing the same change to `package.json`?

Comment (jbalboni) [reply yarn.lock]: The version is within the range specified in package.json, so yarn didn't change it. Yarn won't update it until we get past 5.0.

Comment (erikphansen) [reply yarn.lock]: Oh right, I still just have the habit of explicitly updating `package.json` in cases like this but I know that isn't standard practice.

Just curious: what `yarn` command do you run to fetch the latest and greatest package from NPM that meets what we specify in `package.json`?

Comment (jbalboni) [reply yarn.lock]: I use `yarn upgrade <package name>`, and then when I need to get the absolute most recent, `yarn upgrade <package name> --latest`

File: src/applications/personalization/dashboard/components/DashboardAlert.jsx [hunk 2]
```diff
@@ -30,15 +30,23 @@ const DashboardAlert = ({
   return (
     <div className={classes}>
       <header>
-        {subheadline && <h4>{subheadline}</h4>}
-        <h3>{headline}</h3>
+        {subheadline && (
```
Comment (erikphansen) [inline src/applications/personalization/dashboard/components/DashboardAlert.jsx]: I ran this locally to take a quick look at the `DashboardAlert` with the new markup and it all looks good to me.

**Review by erikphansen — APPROVED** [review]

File: yarn.lock (hunk 1 above)
Comment (erikphansen) [reply yarn.lock]: Cool, that's more efficient than blowing away `node_modules` and rerunning `yarn` :-p

**Review by 1Copenut — APPROVED** [review]
LGTM. I pulled the branch locally and only see one heading best practice warning. That's related to the footer jumping to H4. That's not an issue IMO; the page is organized with semantic headers and we have a way to tailor them as needed now.

**Pull request merged**